### DISCUSSION TITLE
Log4J-Core files in Docker overlay file - vulnerability scanner finding

### DISCUSSION THREAD

**Discussion by scarl3t0** [Security-Onion-Solutions/securityonion#7479] [Unsupported Versions] [answered]
Hello,

Our vulnerability scanner has found an "unacceptable" Log4j-core version on our manager node and a single search node in a Docker overlay file at path:

/var/lib/docker/overlay2/*/diff/usr/share/logstash/logstash-core/lib/jars/log4j-core-2.14.0.jar 

My question is, can I remove this file or is there a fix I can apply to mitigate this finding? Sorry in advance if this is a dumb question - I'm not a Docker expert :) The whole grid has been updated to the most recent SO hotfix release. 

TYIA!

**Comment by scarl3t0** [comment, marked as answer]
As previously thought this was a silly question that I was able to solve on my own. Just for awareness for the group, running docker system prune -a -f seems to have removed the overlays and made the scanners happy. :)

**Reply by dougburks** [reply]
For others who may come across this discussion, this is documented at:
https://docs.securityonion.net/en/2.3/soup.html#security-onion-version-updates
